Formula/pymupdf.rb

```diff
@@ -0,0 +1,47 @@
+class Pymupdf < Formula
+  desc "Python bindings for the PDF toolkit and renderer MuPDF"
+  homepage "https://github.com/pymupdf/PyMuPDF"
+  url "https://files.pythonhosted.org/packages/19/2d/73cb79152442ace5a6f55de17755e7c4c0dbed5ac6180baa1767d6a0e279/PyMuPDF-1.20.0.tar.gz"
+  sha256 "443675ed28dc9be5c9521e17ff9a20299a78b8b94f4c457d7b7aa81899c00ee7"
+  license "AGPL-3.0-only"
+
+  depends_on "freetype" => :build
+  depends_on "swig" => :build
+
+  depends_on "mupdf"
+  depends_on "python@3.9"
```

Comment:
Can this work with 3.10?

Reply:
It can. I just thought that 3.9 is a baseline for libraries at homebrew, because [pyqt also has 3.9](https://github.com/Homebrew/homebrew-core/blob/master/Formula/pyqt.rb#L20) for example.

Reply:
If it can it should.

Reply:
Bumped python version to 3.10